src/content/tutorials/observability/data-exploration/grafana-backups/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+linkTitle: Grafana backups
```

Comment:
I think this belongs to the intranet right?

Reply:
I don't see a reason why customers would create a backup

Reply:
I thought they might want to have some control over it :shrug:

Reply:
It 100% makes sense to me that customers might want to tweak things or back up on demand, BUT

If backups are to be customer configurable, I'd strongly prefer if there's an o11y platform way to configure it so that customers don't need to touch CNPG primitives. We aren't providing CNPG as a service to them, it should just be an internal implementation detail.

Reply:
Ok makes sense to me :+1: 

I'll just keep the change concerning the dashboard creation method then and remove this for now.

Reply:
Thanks :pray: If you want, the backup reference material could still go to internal docs, for other folks working with CNPG

Reply:
yeah was planning on it :)